### DISCUSSION TITLE
Bordered Polyline

### DISCUSSION THREAD

**Discussion by karussell** [openlayers/openlayers#15750] [Q&A] [answered]
Is it possible to give a polyline a different color for the border of it?

**Comment by ahocevar** [comment]
Yes! Just create two styles: a thicker one for the border, and on top a thinner one for the stroke.

**Reply by karussell** [reply]
Nice idea, thanks! Is there an efficient way where I could "duplicate the style" somehow or do I have to create two features?

I currently have:

```js
const borderFeature = new Feature(lineString)
const layer = new VectorLayer({
    source: new VectorSource({
        features: [borderFeature, new Feature(lineString)],
    }),
    style: f => f == borderFeature ? borderStyle : style
})
```

**Comment by ahocevar** [comment, marked as answer]
No need to create another feature. Just return an array of styles instead of a single style from your style function, or configure the layer's style with a static array of two styles.

Change your code to the following, and you're all set:
```js
const layer = new VectorLayer({
    source: new VectorSource({
        features: [new Feature(lineString)],
    }),
    style: [borderStyle, style]
})

**Reply by karussell** [reply]
Ah, nice! Thanks a lot!
